### ISSUE TITLE
Not-an-issue issue

### ISSUE THREAD

**Issue by KoiosLabs** [bofh69/nfc2klipper#4]
This isn't really a problem with your code per-se, but I spent several weeks, and a lot of effort figuring this nonsense out.  
tl;dr: if you have a rpi 5, run sudo rpi-update, or you'll have a bad time

Basically the uart is super buggy on the rpi 5, and I was getting all kinds of weird behaviour with the various nfc libraries that implement the pn532 chipset.  This firmware update solved everything.  I went so far as to go down the rabbit hole of scoping it, and validating the data going back and forth is right, but not being read properly by the pi uarts.

Save your self about 100 hours of banging your head against a wall, and do the firmware update if it doesn't just work out of the box.

Feel free to close this, but you may want to throw a note in the docs somewhere, for those of us dumb enough to run the pi5 :)
